src/server/controllers/movie.js

```diff
@@ -14,13 +14,19 @@ const createMovie = async (req, res) => {
     const { title, description, runtimeMins } = req.body;
 
     try {
-        const token = null;
```

Comment:
You still have your token set as null - how did you manage to get a working results, I wonder